dynamic_rest/prefetch.py

```diff
@@ -205,13 +239,32 @@ def execute(self):
         # TODO: check if queryset already has values() called
         # TODO: use self.fields
         qs = self.queryset._clone()
-        data = list(qs.values())
 
-        self.merge_prefetch(data)
+        has_fastquery = hasattr(self.model, 'use_fastquery')
```

Comment:
So, echoing Ryo's other comment, this should probably be something like `USE_FASTQUERY` to make it clear that it's a constant.